src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge29.java

```diff
@@ -0,0 +1,77 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+@Component
+@Order(29)
+@RestController
+public class Challenge29 extends Challenge {
+
+    @GetMapping("/localStorageString")
+    public String getMyString() {
+        return "ThisIsYourPasswordOfChallenge30";
+    }
+
+    public String solution="ThisIsYourPasswordOfChallenge30";
```

Comment:
Can you have a look at Challenge8 (see https://github.com/OWASP/wrongsecrets/blob/dd657b7c82e7f4ed4a7ef0134a521e10cc96429d/src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge8.java#L24 ) and the controller(https://github.com/OWASP/wrongsecrets/blob/0001ff66ed249becbceffb1d5b3a9b143b2c55e1/src/main/java/org/owasp/wrongsecrets/challenges/ChallengesController.java#L153) and make this random instead?